Misc/NEWS.d/next/Library/2020-12-26-20-30-04.bpo-41781.FibaT_.rst

```diff
@@ -0,0 +1 @@
+Rectify 'chek' typo in Typing.py to 'check'.
```

Comment:
We cloudn't add the `news` if the feature or bugfix doesn't affect outside users.

Reply:
I'll remove this tomorrow, thanks.